--- src/kwidgetsaddons-5.26.0/CMakeLists.txt	2016-09-06 13:23:43.000000000 +0100
+++ src/kwidgetsaddons-5.26.0/CMakeLists.txt	2017-08-03 13:01:59.066114200 +0100
@@ -7,7 +7,7 @@
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
 feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
-
+cmake_policy(SET CMP0053 NEW)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR})
 
 include(KDEInstallDirs)
