--- src/kxmlgui-5.26.0/CMakeLists.txt	2017-07-31 21:43:01.197558600 +0100
+++ src/kxmlgui-5.26.0/CMakeLists.txt	2017-07-31 21:47:21.998236700 +0100
@@ -57,6 +57,9 @@
 find_package(KF5TextWidgets ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5WidgetsAddons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5WindowSystem ${KF5_DEP_VERSION} REQUIRED)
+find_package(KF5Service ${KF5_DEP_VERSION} REQUIRED)
+find_package(KF5Completion ${KF5_DEP_VERSION} REQUIRED)
+find_package(QT5TextToSpeech  REQUIRED)
 find_package(KF5Attica ${KF5_DEP_VERSION})
 set_package_properties(KF5Attica PROPERTIES DESCRIPTION "A Qt library that implements the Open Collaboration Services API"
                        PURPOSE "Support for Get Hot New Stuff in KXMLGUI"
