*** src/zbar/zbar/processor/win.c	Wed Sep 13 17:19:01 2017
--- src/zbar/zbar/processor/win.c	Fri Sep 08 12:27:49 2017
***************
*** 117,128 ****
                                            LPARAM lparam)
  {
      zbar_processor_t *proc =
!         (zbar_processor_t*)GetWindowLongPtr(hwnd, GWL_USERDATA);
      /* initialized during window creation */
      if(message == WM_NCCREATE) {
          proc = ((LPCREATESTRUCT)lparam)->lpCreateParams;
          assert(proc);
!         SetWindowLongPtr(hwnd, GWL_USERDATA, (LONG_PTR)proc);
          proc->display = hwnd;
  
          zbar_window_attach(proc->window, proc->display, proc->xwin);
--- 117,128 ----
                                            LPARAM lparam)
  {
      zbar_processor_t *proc =
!         (zbar_processor_t*)GetWindowLongPtr(hwnd, GWLP_USERDATA);
      /* initialized during window creation */
      if(message == WM_NCCREATE) {
          proc = ((LPCREATESTRUCT)lparam)->lpCreateParams;
          assert(proc);
!         SetWindowLongPtr(hwnd, GWLP_USERDATA, (LONG_PTR)proc);
          proc->display = hwnd;
  
          zbar_window_attach(proc->window, proc->display, proc->xwin);
