--- src/kdeclarative-5.26.0/CMakeLists.txt	2017-08-01 09:29:39.468867600 +0100
+++ src/kdeclarative-5.26.0/CMakeLists.txt	2017-08-01 09:40:59.489028000 +0100
@@ -8,6 +8,10 @@
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
 feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
+if (POLICY CMP0048)
+    cmake_policy(SET CMP0048 OLD)
+endif()
+
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR})
 # Long Windows Path change dir listing char limit.
 set(CMAKE_OBJECT_PATH_MAX 220)
