--- src/kio-5.26.0/CMakeLists.txt	2016-09-06 13:07:20.000000000 +0100
+++ src/kio-5.26.0/CMakeLists.txt	2017-07-31 23:05:27.891569800 +0100
@@ -43,6 +43,7 @@
 find_package(KF5I18n ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5Service ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5DocTools ${KF5_DEP_VERSION})
+find_package(QT5TextToSpeech)
 find_package(KF5Solid ${KF5_DEP_VERSION} REQUIRED) # for kio_trash
 
 if (NOT KIOCORE_ONLY)
