--- src/kio-5.26.0/CMakeLists.txt	2017-07-31 23:08:33.791665500 +0100
+++ src/kio-5.26.0/CMakeLists.orig	2017-07-31 23:12:11.566245600 +0100
@@ -2,6 +2,10 @@
 
 project(KIO)
 
+if (POLICY CMP0048)
+    cmake_policy(SET CMP0048 OLD)
+endif()
+
 include(FeatureSummary)
 find_package(ECM 5.26.0  NO_MODULE)
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
