--- src/solid-5.37.0/CMakeLists.txt	2016-09-06 13:26:53.000000000 +0100
+++ src/solid-5.37.0/CMakeLists.txt	2017-08-05 14:30:53.480197100 +0100
@@ -8,6 +8,9 @@
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://projects.kde.org/projects/kdesupport/extra-cmake-modules")
 feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
+if (POLICY CMP0053)
+    cmake_policy(SET CMP0053 OLD)
+endif()
 
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
